docs/naming.md

```diff
@@ -1,86 +1,169 @@
-# Imagery naming conventions
+# Imagery Dataset Naming Conventions
 
-Imagery is named according to the majority region that imagery covers while also adhering to a few naming conventions.
+Imagery dataset titles and S3 paths are constructed from metadata about each imagery survey so that they will be consistent and human readable. Imagery is stored according to the main region that each dataset covers. Datasets that cover 2 or more full regions may instead be stored under `new-zealand`.
```

Comment:
```suggestion
Imagery dataset titles and S3 paths are constructed from metadata about each imagery survey so that they will be consistent and human readable. Imagery is stored according to the main region that each dataset covers. Datasets that cover two or more full regions may instead be stored under `new-zealand`.
```

In other places, numbers are written in letters (one instead of 1)